app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -991,9 +990,9 @@ private Pokemon interpretWhichPokemonUserInput() {
      * screenshot, and then deletes the screenshot.
      */
     private void deleteScreenShotIfRequired() {
-        if (batterySaver && !ssFile.isEmpty()) {
```

Comment:
@sarav This is the only call to `isEmpty` and gets removed here. So I'm not sure about your message—and if it's something else that is crashing in `deleteScreenShotIfRequired`, I'm not sure what.

> I'm still getting a crash with Google Now on Tap share on isEmpty() in deleteScreenShotIfRequired() when I click "checkIV"